pyproject.toml

```diff
@@ -19,7 +19,7 @@ packages = [
 python = "^3.10"
 pyteal = "~0.23.0"
 py-algorand-sdk = ">=2.0.0"
-algokit-utils = "^1.0.0"
+algokit-utils = "^1.0.2b7"
```

Comment:
Will do a proper release before merging this PR

Reply:
I've approved but will hold off merging until this one is resolved

Reply:
Updated to 1.0.2 proper